src/snowflake/connector/auth/_auth.py

```diff
@@ -166,10 +168,14 @@ def authenticate(
         mfa_callback: Callable[[], None] | None = None,
         password_callback: Callable[[], str] | None = None,
         session_parameters: dict[Any, Any] | None = None,
-        timeout: int = 120,
+        timeout: int
+        | None = None,  # max time waiting for MFA response, currently unused
```

Comment:
```suggestion
        timeout: int | None = None,  # max time waiting for MFA response, currently unused
```

Reply:
Precommit checks keep reformatting that line automatically D:<